### ISSUE TITLE
Milestones and backlog

### ISSUE THREAD

**Issue by OksanaShulha** [HackYourFutureBelgium/class-13-14#109]
###Hi there!
I am kindly asking you to explain how milestones could be created based on the user stories in our backlog - one milestone per user story? I look at my user story in Group 5 and I can not understand how it is suppose to be....
https://github.com/IrinaSing/Loruki-5/tree/main/planning

**Comment by colevandersWands** [comment]
Hey @OksanaShulha, the question is open!  Could you include a link to your group 5 project planning for more context?

**Comment by OksanaShulha** [comment]
Sorry for duplicating it twice, I'm writing an issue from the phone and it is not very convenient)

**Comment by OksanaShulha** [comment]
OK, I am going include!

**Comment by colevandersWands** [comment]
no problem! thanks for the link

**Comment by snicoll** [comment]
We've discussed this one in class today. A milestone can be seen as a group of issues that represents an objective. It can be time-based, version-based or both. A milestone is very useful to see progress (how much issues are closed and how much are still left).

As for naming, the usual "naming is hard". We discussed gathering as a team and find a pattern and or naming. There isn't a single answer but the key is that the team is comfortable with the naming.

See also https://guides.github.com/features/issues/
